package.json

```diff
@@ -13,8 +13,9 @@
         "web"
     ],
     "scripts": {
-        "start": "webpack --mode development --watch",
-        "build": "webpack --mode production",
+        "start": "npm run lint && tsc",
+        "build": "npm run lint && tsc",
+        "lint": "eslint --ext .ts src/*",
```

Comment:
this should probably be the same command as in lint-staged: `eslint -c ./.eslintrc.js --fix`